Yank/experiment.py

```diff
@@ -2143,17 +2176,24 @@ def _build_experiment(self, experiment_path, experiment):
             solvent_dsl = 'auto'  # Topography uses common solvent resnames.
         logger.debug('DSL string for the solvent: "{}"'.format(solvent_dsl))
 
-        # Determine complex and solvent phase solvents
+        # Determine complex and solvent phase solvents while also getting regions
+        db_system = self._db.systems[system_id]
```

Comment:
Could you maybe call this `system_description`? I think that's the common variable naming convention in the class.